lib/flutter_crashlytics.dart

```diff
@@ -66,4 +67,52 @@ class FlutterCrashlytics {
     return await _channel.invokeMethod(
         'setUserInfo', {"id": identifier, "email": email, "name": name});
   }
+
+  /// Reports an Exception to Craslytics together with the stacktrace.
+  /// Both fields are mandatory.
+  ///
+  /// As Errors should not be cougth those are not supported.
+  ///
+  /// ```dart
+  /// try {
+  ///     // Code throwing en exception
+  /// } on Exception catch (e, s) {
+  ///     FlutterCrashlytics().logException(e, s);
+  /// }
+  /// ```
+  Future<void> logException(Exception exception, StackTrace stack) async {
```

Comment:
I'm ok for logException, but a logError would be useful too, because sometime you'll have the case of using third party sdk/libraries that throw Error and want to catch them. Can you add it too please ?

Reply:
Added